tools/toolchains/bazel4_local_config_cc/BUILD

```diff
@@ -89,7 +89,7 @@ cc_toolchain_suite(
         compiler = "compiler",
         cpu = arch,
         cxx_builtin_include_directories = [
-            "/Applications/",
+            "/",
```

Comment:
This is a temporary change to workaround a problem here (https://github.com/bazelbuild/bazel/issues/14346) with inclusion validation logic. We won't need this bazel5 toolchain stuff once we update to the next bazel5.0 release candidate. We need the following change (https://github.com/bazelbuild/bazel/commit/005361c895da334beb873901e93aff06d180256e)  which the bazel release team accidentally omitted from the rc3 cut. see https://github.com/bazelbuild/bazel/pull/14440#issuecomment-996394207 as well.

The current bazel5 release candidate is rc3, which can be determined by checking the bazel-discuss email list: https://groups.google.com/g/bazel-discuss/